emailApp/smtp/smtp.js

```diff
@@ -0,0 +1,42 @@
+/* SmtpJS.com - v3.0.0 */
```

Comment:
Please add file description on the top of the corresponding file.

Reply:
done